alma/sfcc-cartridge
# Remove restriction to filter out deferred payments from In-Page

## Reason for change



[Linear task](https://linear.app/almapay/issue/ECOM-1464/[🧩-sfcc]-remove-restriction-to-filter-out-deferred-payments-from-in)

### Code changes

Edit the function isAvailableForInpage to change the condition to enablement of payment method

Update unit tests



### How to test

_As a reviewer, you are encouraged to test the PR locally._

Go to the checkout page 
Pay with a method deferred and see if the in page is displayed



### Checklist for authors and reviewers




### Non applicable
